tvb_contrib/tvb/contrib/tests/cosimulation/parallel/double_proxy_precision_complex_test.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
```

Comment:
similar comments as the other test, watch numbers and precision, use numpy testing functions etc